docs/changes.xml

```diff
@@ -37,6 +37,12 @@ compatibility with GCC 12.
 </para>
 </change>
 
+<change type="feature">
+<para>
+supporting UNIX sockets in address matching
```

Comment:
The dot at the end of sentence is missing.

Also I usually try to estimate subjectively the amount of users affected or importance of the feature/fix for them and sort relatively to that estimation. In this case, this feature seems more important than support for a not-yet-released compiler version (so it should go above).

Also 2, it makes sense to consult with Artem about the best wording. Commit log for devs, change log for users. The current wording is good enough, but I have some doubts that many users will understand what "address matching" is meant here. Maybe it's worth to describe specifically, that this is about address matching in routing and "client_ip" options.

Reply:
> The dot at the end of sentence is missing.

D'oh.

> 
> Also I usually try to estimate subjectively the amount of users affected or importance of the feature/fix for them and sort relatively to that estimation. In this case, this feature seems more important than support for a not-yet-released compiler version (so it should go above).

Makes sense.

> 
> Also 2, it makes sense to consult with Artem about the best wording. Commit log for devs, change log for users. The current wording is good enough, but I have some doubts that many users will understand what "address matching" is meant here. Maybe it's worth to describe specifically, that this is about address matching in routing and "client_ip" options.

Yes, and yes, I'll add those details regarding client_ip.